src/itensor.jl

```diff
@@ -2129,28 +1702,12 @@ function indpairs(T::ITensor; plev::Pair{Int,Int}=0 => 1, tags::Pair=ts"" => ts"
 end
 
 """
-    transpose(T::ITensor)
-
-Treating an ITensor as a map from a set of indices
-of prime level 0 to a matching set of indices but
-of prime level 1 
-[for example: (i,j,k,...) -> (j',i',k',...)]
-return the ITensor which is the transpose of this map.
-"""
-transpose(T::ITensor) = swapprime(T, 0 => 1)
-
-"""
-    ishermitian(T::ITensor; kwargs...)
-
-Test whether an ITensor is a Hermitian operator,
-that is whether taking `dag` of the ITensor and
-transposing its indices returns numerically 
-the same ITensor.
+      tr(T::ITensor; kwargs)
+Trance of an ITensor over a pair of indices
```

Comment:
I believe that there needs to be a line separating the function signature and the description for it to format correctly by Documentor.

Also "Trance" -> "Trace".

Reply:
Ahh got it, thanks!